dhis-2/dhis-api/src/main/java/org/hisp/dhis/webapi/controller/event/mapper/OrderParamsHelper.java

```diff
@@ -64,16 +64,20 @@
         Map<String, TrackedEntityAttribute> attributes )
     {
         List<String> errors = new ArrayList<>();
-        if ( orderParams != null && !orderParams.isEmpty() )
+
+        if ( orderParams == null || orderParams.isEmpty() )
```

Comment:
if orderParams.isEmpty() the later loop will not do anything, so this check is redundant